app/controllers/v0/sign_in_controller.rb

```diff
@@ -320,19 +317,17 @@ def delete_cookies
 
     def auth_service(type)
       case type
+      when 'mocked_authentication'
+      SignIn::MockedAuthentication::Service.new
```

Comment:
This should be the `mocked_authentication_auth_service` method that you created below (or `vamock_auth_service` if you change it)